src/components/uploader/js/Uploader.js

```diff
@@ -200,20 +191,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.uploader.updateTotalProgress = function (that) {
-        // Preservered for backwards compatibility, should be refactored post v1.5
         var batch = that.queue.currentBatch;
         var totalPercent = fluid.uploader.derivePercent(batch.totalBytesUploaded, batch.totalBytes);
-        var numFilesInBatch = batch.files.length;
-        var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
-
         var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
 
         var totalProgressStr = fluid.stringTemplate(that.options.strings.progress.totalProgressLabel, {
-            curFileN: batch.fileIdx,
-            totalFilesN: numFilesInBatch,
-            fileLabel: fileLabelStr,
-            currBytes: fluid.uploader.formatFileSize(batch.totalBytesUploaded),
```

Comment:
My thought is that we don't use them so it becomes an added maintenance burden, plus a higher chance of breaking without us catching it. Also, I'd like to reduce the file size and javascript processing as much as possible. The more we keep building up the larger these inflate over time. I totally agree we should keep this between major releases, but I'd like to take the opportunity at a major release trim what we can.